src/Geosuggest.tsx

```diff
@@ -100,6 +105,9 @@ export default class extends React.Component<IProps, IState> {
     this.onSuggestNoResults = this.onSuggestNoResults.bind(this);
     this.hideSuggests = this.hideSuggests.bind(this);
     this.selectSuggest = this.selectSuggest.bind(this);
+    this.listId = `geosuggest__list_${Math.random()
```

Comment:
Shall we make it possible to pass a list ID, and default to this value if none is passed?

Reply:
This functionality was not provided by geosuggest before, `<ul>`, there were not an id's on UL before, so I decided to live it as it is, without adding new functionality